server/db/controllers/User.js

```diff
@@ -0,0 +1,85 @@
+import jwt from 'jsonwebtoken';
+import moment from 'moment';
+import uuidv4 from 'uuid/v4';
+import verification_code from 'generate-sms-verification-code';
+import ST from '../../helpers/status';
+import MSG from '../../helpers/res_messages';
+import auth from '../middleware/auth' 
+import { CREATE_USER_TABLE, CREATE_USER ,VERIFIE_USER, GET_USER, GET_VERIFICATION, CREATE_VERFICATION,DELETE_VERIFICATION} from '../helpers/query';
+import connection from '../connection/connection';
+import joi from 'joi';
+import Helper from '../../helpers/Helper';
+import validation from '../../helpers/validation';
+import db from '../db';
+const pool = connection.getPoolConnection();
+class User {
+     constructor(){ 
+     }
```

Comment:
Why do you have an empty constructor for this class?

Reply:
Because of this file export the instance of the class user. 
and for the instance definition, I don't parse anything.

Reply:
In that case, do you really need to explicitly define the empty constructor?